### PR TITLE
fix mobile header

### PR DESCRIPTION
**Pull request by lislis**
part of #519 

screenshot-2017-11-7 rorganize it find a rails girls study group 2

Logo, main-nav and profile-nav are now visible and usable on every screen size

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,14 +5,13 @@
   <title>RoRganize.it | Find a Rails Girls Study Group</title>
   <%=  stylesheet_link_tag (params[:redesign] ?  'redesign' :  'application'), media: "all" %>
   <%= csrf_meta_tags %>
-  <link href='//fonts.googleapis.com/css?family=Pacifico' rel='stylesheet' type='text/css'>
```
Comment (sareg0) [inline app/views/layouts/application.html.erb]: Nice!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,14 +5,13 @@
   <title>RoRganize.it | Find a Rails Girls Study Group</title>
   <%=  stylesheet_link_tag (params[:redesign] ?  'redesign' :  'application'), media: "all" %>
```
Comment (sareg0) [inline app/views/layouts/application.html.erb]: Do you think we can also get rid of this? We can start a new issue for it, so as not to bloat this one.

**Comment by sareg0** [comment]
Thanks soooo much for this. Since there is a fair bit of re-organization of the files, could you point out the fixes that solve the particular UI issues?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (lislis) [reply app/views/layouts/application.html.erb]: Ah, good catch! I can definitely remove that

File: app/assets/stylesheets/custom/_nav.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
+// Header + Navigation
+
+$navbar-height-reduced: 78px; // some weirdness on smaller screens
+
+nav.navbar {
```
Comment (lislis) [inline app/assets/stylesheets/custom/_nav.scss]: @sareg0 most things happen here. I changed the flex-order of the elements that appear in the header to better fit smaller screens. Some elements were also hidden on mobile devices which are now always visible

**Comment by lislis** [comment]
@sareg0 sure thing! I left a comment in the files :)

**Comment by sareg0** [comment]
LGTM

**Pull request merged**